.github/workflows/update-offsets.yml

```diff
@@ -15,7 +14,7 @@ jobs:
     - name: "Checkout repo"
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
       with:
-        persist-credentials: false
+        persist-credentials: true
```

Comment:
the zizmor checker seems to be ok as long as it's explicitly opted into (which makes sense to avoid people using the default not knowing better)